docs/source/private-data/private-data.md

```diff
@@ -347,6 +348,346 @@ submitted a letter of credit, prior to the execution of the transfer chaincode.
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org1MSP"
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export CORE_PEER_ADDRESS=localhost:7051
+```
+The environment variables also specify the endpoint information of the Org1 peer to submit requests.
+
+#### Set the environment variables to operate as Org2
+
+Now that we have one terminal that we can operate as Org1, open a new terminal for Org2. Make sure that this terminal is also operating from the `test-network` directory. Set the following environment variables to operate as the Org2 admin:
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+export FABRIC_CFG_PATH=$PWD/../config/
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org2MSP"
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+You will need switch between the two terminals as you go through the tutorial.
+
+### Deploy the chaincode
+
+Now that we can operate as both organizations, we need install the private asset transfer smart contract on the peers of Org1 and Org2, and deploy the chaincode to the channel approving and committing the chaincode definition.
+
+#### Install and approve the chaincode as Org1
+
+Open the Org1 terminal. Run the following command to package the private asset transfer chaincode:
+```
+peer lifecycle chaincode package assets_transfer.tar.gz --path ../asset-transfer-secured-agreement/chaincode-go --lang golang --label assets_transfer_1
+```
+
+The command creates a chaincode package named `assets_transfer.tar.gz`. We can now install this package on the Org1 peer:
+```
+peer lifecycle chaincode install assets_transfer.tar.gz
+```
+
+You will need the chaincode package ID in order to approve the chaincode definition. You can find the package ID by querying your peer:
+```
+peer lifecycle chaincode queryinstalled
+```
+Save the package ID as an environment variable. The package ID will not be the same for all users, so need to use the result that was returned by the previous command:
+```
+export PACKAGE_ID=assets_transfer_1:2a585633baa0a6ba0019965ac40d6f188194c50df1015010b080ef6ba426d266
+```
+You can now approve the chaincode as Org1:
+```
+peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name assets_transfer --version 1 --package-id $PACKAGE_ID --sequence 1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --signature-policy "OR('Org1MSP.peer','Org2MSP.peer')"
```

Comment:
A more explicit chaincode name in all the commands will ensure that all our asset_transfer_* samples can be installed at same time
```suggestion
peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name assets_transfer_secure --version 1 --package-id $PACKAGE_ID --sequence 1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --signature-policy "OR('Org1MSP.peer','Org2MSP.peer')"
```

btw, once we have new short names in test-network, per https://github.com/hyperledger/fabric-samples/pull/235 , we should add a `QuickInstall` instruction using the chaincode short name called `secure`. I think having the detailed steps like this is still useful for devops to do custom steps.

Reply:
🛠️ wording; Good idea, we can add `QuickInstall` in a follow-up pr.